src/main/java/com/Saurabh/GitHubAiAPI/StringManipulation.java

```diff
@@ -33,6 +33,14 @@ public static void main(String[] args) {
             stringArrayList.add(str);
         }
```

Comment:
Consider making the string to be replaced ('India') and the string to replace it with ('America') as parameters to the method. This would make the method more flexible and reusable.